### PR TITLE
feat: add coverage metrics

### PR DESCRIPTION
**Pull request by childish-sambino**
Coverage here: https://sonarcloud.io/summary/new_code?id=twilio_twilio-oai-generator&branch=coverage

### CODE REVIEW

File: examples/go-client/go.mod [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module go-client
```
Comment (beebzz) [inline examples/go-client/go.mod:1]: How come this go-client for Twilio is being baked into this SonarCloud PR?

File: prism.sh [hunk 2]
```diff
@@ -5,25 +5,25 @@ make install
 
 API_SPEC=examples/twilio_api_v2010.yaml
 
-OUT_DIR=examples/go/rest
-rm -rf $OUT_DIR
+OUT_DIR=examples/go-client/helper/rest
+find $OUT_DIR ! -name "*_test.go" -type f -delete
```
Comment (rakatyal) [inline prism.sh:9]: Just for the sake of curiosity, why the change from deleting the entire directory to the test files?

**Review by rakatyal — APPROVED** [review]
